test/benchee_test.exs

```diff
@@ -418,6 +422,35 @@ defmodule BencheeTest do
     assert length(occurences) == 3
   end
 
+  test "inputs can also be a list of 2-tuples" do
+    output =
+      capture_io(fn ->
+        map_fun = fn i -> [i, i * i] end
+
+        inputs = [
+          inputs: [
+            {"small list", Enum.to_list(1..100)},
+            {"medium list", Enum.to_list(1..1_000)},
+            {"bigger list", Enum.to_list(1..10_000)}
+          ]
+        ]
+
+        configuration = Keyword.merge(@test_configuration, inputs)
+
+        Benchee.run(
+          %{
+            "flat_map" => fn input -> Enum.flat_map(input, map_fun) end,
+            "map.flatten" => fn input -> input |> Enum.map(map_fun) |> List.flatten() end
+          },
+          configuration
+        )
+      end)
+
+    assert String.contains?(output, ["small list", "medium list", "bigger list"])
+    occurences = Regex.scan(body_regex("flat_map"), output)
+    assert length(occurences) == 3
```

Comment:
:ok_hand: 

We could actually check on the global level here if order is retained if we wrote a regex that asserts their order. What do you think?